src/Resources/contao/library/Contao/Encryption.php

```diff
@@ -157,7 +166,12 @@ protected static function initialize()
 			throw new \Exception('Error initializing encryption module');
 		}
 
-		if (\Config::get('encryptionKey') == '')
+		/** @var KernelInterface $kernel */
+		global $kernel;
+
+		$container = $kernel->getContainer();
+
+		if ($container->getParameter('framework.secret') == '')
```

Comment:
That's not what I mean. It would throw an `ParameterNotFoundException` if the parameter is not set, which should never happen. However, I can set the secret to actually be an empty string, which this check would then catch.